### PR TITLE
Update number of supported DEXes in About page

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
| Before | After |
| --- | --- |
| image | image |

### CODE REVIEW

File: src/pages/About/AboutKyberSwap.tsx [hunk 1]
```diff
@@ -150,7 +148,7 @@ export const KSStatistic = () => {
         <ForTraderInfoRow>
           <ForTraderInfoCell>
             <Text fontWeight="600" fontSize="24px">
-              {MAINNET_NETWORKS.length}
```
Comment (namgold) [inline src/pages/About/AboutKyberSwap.tsx:153]: {MAINNET_NETWORKS.length-1}+

File: src/pages/About/AboutKyberSwap.tsx [hunk 2]
```diff
@@ -150,7 +148,7 @@ export const KSStatistic = () => {
...
+              12+
```
Comment (namgold) [inline src/pages/About/AboutKyberSwap.tsx:151]:
```suggestion
              {MAINNET_NETWORKS.length-1}+
```

Comment (namgold) [reply src/pages/About/AboutKyberSwap.tsx:151]: When adding new chain, everyone gonna miss places like this. Keep it reference to network const to keep it automatically updated.

File: src/constants/dexes.ts [hunk 3]
```diff
@@ -277,6 +277,10 @@ export const dexListConfig: DexList = {
     name: 'Velodrome',
     icon: 'https://s2.coinmarketcap.com/static/img/coins/64x64/20435.png',
   },
+  dystopia: {
+    name: 'Dystopia',
+    icon: 'https://storage.googleapis.com/ks-setting-1d682dca/7693d113-5321-414c-a7c7-11d501b49379.png',
+  },
```
Comment (namgold) [inline src/constants/dexes.ts:283]: No need

File: src/pages/About/AboutKyberSwap.tsx (hunk 2 above)
Comment (hungdoansy) [reply src/pages/About/AboutKyberSwap.tsx:151]: Still hard for others to know why minus 1

File: src/constants/dexes.ts (hunk 3 above)
Comment (namgold) [reply src/constants/dexes.ts:283]: You can refactor to remove this file, or most line of this file

File: src/pages/About/AboutKyberSwap.tsx (hunk 2 above)
Comment (namgold) [reply src/pages/About/AboutKyberSwap.tsx:151]: write down your reason to a comment next to it

File: src/constants/dexes.ts (hunk 3 above)
Comment (hungdoansy) [reply src/constants/dexes.ts]: Ok. I removed it

File: src/pages/About/AboutKyberSwap.tsx (hunk 2 above)
Comment (hungdoansy) [reply src/pages/About/AboutKyberSwap.tsx:151]: I don't have a reason...

Comment (namgold) [reply src/pages/About/AboutKyberSwap.tsx:151]: then no need. I see that `{...- 1}+` is very meanful

Comment (hungdoansy) [reply src/pages/About/AboutKyberSwap.tsx]: Ok. Done now

**Review by namgold — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**